app/errors.py

```diff
@@ -0,0 +1,18 @@
+class Error(Exception):
```

Comment:
This class is unnecessary. Remove it and inherit `VaccineError` and `NotWearingMaskError` directly from `Exception`.